### ISSUE TITLE
Generated Link from the AppFlyer doesn't open the installed app (iOS)

### ISSUE THREAD

**Issue by metinjakupi** [AppsFlyerSDK/appsflyer-capacitor-plugin#118]
# Report

## Plugin Version

ℹ Please replace these two lines with the plugin version.
6.13.1

## On what Platform are you having the issue?

ℹ Please replace these two lines with the SDK version.
iOS

## What did you do?

ℹ Please replace these two lines with what you did.  
npm install appsflyer-capacitor-plugin 
npx cap sync  

## What did you expect to happen?

ℹ Please replace these two lines with what you expected to happen.  
The installed app to be opened

## What happened instead?
It opens the App Store

**Comment by OskarLebuda** [comment]
Same issue here. @metinjakupi did you resolved that?

**Comment by metinjakupi** [comment]
@OskarLebuda Yeah, I resolved it - I simply stopped developing Capacitor apps! 😅
